### PR TITLE
Feat/about news --database seeding

### PR DESCRIPTION
**Pull request by morgangrace11**

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    type: Sequelize.INTEGER,
+    primaryKey: true,
+    autoIncrement: true
+  },
+  description: Sequelize.STRING(2000),
+  ceo: Sequelize.STRING,
+  employees: Sequelize.INTEGER,
+  headquarters: Sequelize.STRING,
+  founded: Sequelize.INTEGER,
+  marketCap: Sequelize.INTEGER,
```
Comment (AKhoo) [inline database/index.js]: Does marketCap, P/E and div yield need to be a Float? (ie support decimals)?

File: database/index.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      pictureUrl: faker.image.imageUrl(),
+      name: faker.company.companyName(),
+      title: faker.lorem.sentence(),
+      views: faker.random.number(),
+      description: faker.lorem.sentences()
+    })
+  }
+}
+
+fakeNews();
```
Comment (AKhoo) [inline database/index.js]: How does the relationship between News and About get created?

File: README.md [hunk 3]
```diff
@@ -19,6 +19,14 @@
 
   Some usage instructions
 
+    -npm run database
+      -creates the database
+    -npm run start
+      -starts your server/creates your table
+    -npm run seed
+      -seeds your database with fake data
+
```
Comment (AKhoo) [inline README.md]: Great idea updating the Readme regularly! I'm gonna do that too :)

File: database/index.js (hunk 1 above)
Comment (morgangrace11) [reply database/index.js]: From my understanding with robin It just need to be random numbers but I can change the data type to some financial numbers and decimals and such!

File: package.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    "mysql2": "^1.6.4",
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0",
+    "sequelize": "^4.42.0",
+    "style-loader": "^0.23.1",
+    "webpack": "^4.28.0",
+    "webpack-cli": "^3.1.2"
+  },
+  "devDependencies": {
+    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
```
Comment (therobinkim) [inline package.json]: imo probably better to use airbnb's ruleset for `eslint`

File: README.md (hunk 3 above)
Comment (therobinkim) [reply README.md]: +1!

File: .nvmrc [hunk 5]
```diff
@@ -0,0 +1 @@
+v6.13.0
```
Comment (therobinkim) [inline .nvmrc]: use node 10, not node 6!

Comment (morgangrace11) [reply .nvmrc]: Okay will change it!

File: package.json (hunk 4 above)
Comment (morgangrace11) [reply package.json]: I'll look into that, thanks Robin!

**Pull request merged**